src/server/game/Battlefield/Zones/BattlefieldWG.cpp

```diff
@@ -1550,9 +1549,9 @@ void WintergraspWorkshop::UpdateGraveyardAndWorkshop()
         GiveControlTo(_wg->GetDefenderTeam(), true);
 }
 
-void WintergraspWorkshop::FillInitialWorldStates(WorldPacket& data)
+void WintergraspWorkshop::FillInitialWorldStates(WorldPackets::WorldState::InitWorldStates& packet)
 {
-    data << uint32(_staticInfo->WorldStateId) << uint32(_state);
+    packet.Worldstates.push_back({ uint32(_staticInfo->WorldStateId), int32(_state) });
```

Comment:
Yes but first verify that it compiles with a single emplace_back.
If you need help doing a regex to change all of them do say, please don't do it manually :P